src/app/application/components/app-collection/AppCollectionActions.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
Any reason not to inline this inside AppCollection?

Reply:
I suppose because I figure that sometimes an AppCollection might not have any actions at all.

i.e. I don't want to overload AppCollections too much. What if we have an AppCollection that doesn't have a dropdown, but instead has a single button? Or the actions has a dropdown and a checkbox, or possibly other things I haven't thought of?